pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```diff
@@ -0,0 +1,44 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkg-config, glib, go, gtk3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+  
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+  
+  nativeBuildInputs = [ pkg-config go ];
+  
+  buildInputs = [ glib gtk3 libhandy ];
+
+  desktopFile = makeDesktopItem {
+    name = pname;
+    desktopName = "gtkcord";
+    exec = "gtkcord3";
+    icon = "gtkcord3";
+    categories = "GTK;GNOME;Chat;";
+  };
+
+  preFixup = ''
```

Comment:
This might be better as `postInstall` since it is installing files not setting up anything for fixup.